### ISSUE TITLE
Add a flag to force `qri` to use the first line of CSV as field names.

### ISSUE THREAD

**Issue by feep** [qri-io/qri#1318]
`qri` does a good job at inferring whether the first row is a header row.

But it would be nice to be able to tell it explicitly. Especially in cases when it guesses wrong.

**Comment by dustmop** [comment]
Agreed. Would be nice to have a flag like `qri save --format.config.header-row true` or something.

**Comment by feep** [comment]
https://github.com/qri-io/dataset/blob/a8aa56636bdad1ef9170936627ad5c33a7fbebbf/detect/determineFields.go#L82
